### PR TITLE
Added DEX execution support for Lstm & Rnn

### PR DESCRIPTION
**Pull request by pruthviIntel**

### CODE REVIEW

**Comment by pruthviIntel** [comment]
@jbobba addressed your comments.

**Comment by pruthviIntel** [comment]
@jbobba how about now? ^^

File: src/ngraph/runtime/cpu/mkldnn_emitter.hpp [hunk 1]
```diff
@@ -391,6 +392,85 @@ namespace ngraph
                                                 const mkldnn::memory::desc& dweights_desc,
                                                 const double eps);
 
+                template <typename OP>
+                size_t build_rnn(const ngraph::Node* node,
```
Comment (jbobba) [inline src/ngraph/runtime/cpu/mkldnn_emitter.hpp]: looks good. but dont see a need for templatized method.

Comment (pruthviIntel) [reply src/ngraph/runtime/cpu/mkldnn_emitter.hpp]: ya but `OP` can be either `ngraph::op::Lstm` or `ngraph::op::Rnn` so I templatized it

**Pull request merged**